app/javascript/gobierto_data/webapp/components/SidebarCategories.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="pure-u-1 pure-u-lg-4-4">
+    <aside
+      v-if="checkFilters"
+      class="gobierto-data-filters"
+    >
+      <div
+        v-for="filter in filters"
+        :key="filter.title"
+        :class="!filter.key ? 'gobierto-data-filters-element-no-margin' : ''"
+        class="gobierto-data-filters-element"
+      >
+        <template v-if="filter.type === 'vocabulary_options'">
+          <div>
+            <BlockHeader
+              :title="filter.title"
+              :label-alt="filter.isEverythingChecked"
+              :class="filter.key ? '' : 'gobierto-filter-rotate-icon'"
+              see-link
+              @toggle="e => filter.key = !filter.key"
+              @select-all="e => selectAllCheckbox({ ...e, filter })"
+            />
+          </div>
+          <Checkbox
+            v-for="option in filter.options"
+            v-show="filter.key"
+            :id="option.id"
+            :key="option.id"
+            :title="option.title"
+            :checked="option.isOptionChecked"
+            :counter="option.counter"
+            @checkbox-change="e => sendCheckboxStatus({ ...e, filter })"
+          />
+        </template>
+      </div>
+    </aside>
+  </div>
+</template>
+<script>
+import { BlockHeader, Checkbox } from "lib/vue-components";
+export default {
+  name: "SidebarCategories",
+  components: {
+    BlockHeader,
+    Checkbox
+  },
+  props: {
+    activeTab: {
+      type: Number,
+      default: 0
+    },
+    filters: {
+      type: Array,
+      default: () => []
+    }
+  },
+  data() {
+    return {
+      labelSets: "",
+      labelQueries: "",
+      labelCategories: ""
+    }
+  },
+  computed: {
+    checkFilters() {
+      return this.filters[0].count >= 1 || this.filters[1].count >= 1 ? true : false;
```

Comment:
Even better:
```suggestion
if (!this.filters.length) return false
return ((this.filters[0] && (this.filters[0] || {}).count >= 1)
  || (this.filters[1] && (this.filters[1] || {}).count >= 1));
```

Everything may fail.